nrf52840/blinky/arduino/fast/fast.ino

```diff
@@ -0,0 +1,37 @@
+#include <Arduino.h>
+#include <Notecard.h>
+
+#define serialDebugOut Serial
+
+Notecard notecard;
+
+void setup()
+{
+    pinMode(LED_BUILTIN, OUTPUT);
+    Serial.begin(115200);
+
+    serialDebugOut.begin(115200);
+    notecard.setDebugOutputStream(serialDebugOut);
+
+    Wire.begin();
+    notecard.begin();
+
+    J *req = notecard.newRequest("hub.set");
+    JAddStringToObject(req, "mode", "continuous");
+    JAddStringToObject(req, "sn", "blinky-arduino-fast");
+    notecard.sendRequest(req);
+
+    req = notecard.newRequest("card.dfu");
+    JAddStringToObject(req, "name", "nrf52");
+    JAddBoolToObject(req, "on", true);
+    notecard.sendRequest(req);
```

Comment:
I still view this as dangerous, because if this code were flashed to anything other than an `nRF52` device, then it would render Outboard DFU unusable.

Reply:
It's the pattern used in other examples too. I can add a compile time conditional just in case someone tries to compile for a different platform. 

In the long run, I feel this should be something the SDK takes care of. The developer should just make a call to say `enableOutboardFirmwareUpdates()` and the SDK takes care of getting the names right based on the platform being compiled for. This will make the code more portable.